README.md

```diff
@@ -47,8 +51,9 @@ The only option is `--silent`, which suppresses all detailed warnings. Regardles
 ```python
 import fastjsonschema
 import json
+import requests
 
-schema = json.load(open("hif_schema_v0.1.0.json","r"))
+schema = requests.get("https://raw.githubusercontent.com/pszufe/HIF_validators/main/schemas/hif_schema_v0.1.0.json").json()
```

Comment:
I think we need to keep the local link so that GH Actions can test the schema changes

Reply:
I don't thing that GitHub actions is testing the README so my thought was to have a code snippet that users can run without cloning the repo.

Reply:
My bad haha. LGTM.